.env.example

```
@@ -1,3 +1,5 @@
 VITE_UPSTAGE_API_KEY=up_*********KwzE
 VITE_UPSTAGE_SYSTEM_PROMPT="You are a helpful assistant."
-VITE_API_BASE_URL=https://api.example.com
\ No newline at end of file
+VITE_API_BASE_URL=https://api.example.com
+VITE_GOOGLE_CLIENT_ID=YOUR_GOOGLE_CLIENT_ID
+VITE_GOOGLE_CLIENT_SECRET=YOUR_GOOGLE_CLIENT_SECRET
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)